### PR TITLE
Update syntax to be more aligned with es6

### PR DESCRIPTION
**Pull request by zzarcon**
A little syntax changes to be more ES6

### CODE REVIEW

File: app/services/ember-devtools.js [hunk 1]
```diff
@@ -143,19 +137,19 @@ export default Service.extend({
     var skipGlobalize = this.constructor.skipGlobalize;
     if (skipGlobalize === null) {
       skipGlobalize = this.constructor.skipGlobalize = props.filter(function(prop) {
-        return !Ember.isNone(self.global[prop]);
-      });
+        return !Ember.isNone(this.global[prop]);
+      }, this);
```
Comment (aexmachina) [inline app/services/ember-devtools.js]:
```js
skipGlobalize = this.constructor.skipGlobalize = props.filter(prop => !Ember.isNone(this.global[prop]));
```

File: app/services/ember-devtools.js [hunk 2]
```diff
@@ -143,19 +137,19 @@ export default Service.extend({
...
       if (typeof prop === 'function') {
         prop = function() {
-          return self[name].apply(self, arguments);
+          return this[name].apply(this, arguments);
         };
       }
-      self.global[name] = prop;
-    });
+      this.global[name] = prop;
+    }, this);
```
Comment (aexmachina) [inline app/services/ember-devtools.js]: Prefer fat arrow rather than specifying the receiver.

**Comment by aexmachina** [comment]
Nice one! Thanks.

File: app/services/ember-devtools.js (hunk 1 above)
Comment (zzarcon) [inline app/services/ember-devtools.js]: Nice one!

File: app/services/ember-devtools.js (hunk 2 above)
Comment (zzarcon) [inline app/services/ember-devtools.js]: 100% agree

**Comment by zzarcon** [comment]
Changes done :shipit:

**Comment by aexmachina** [comment]
Published v4.3.0

**Pull request merged**